style.css

```
@@ -494,13 +461,15 @@ body[data-bg="wallpaper"] .tooltip-text {
 	box-sizing: border-box;
-	z-index: 3; /* This should be higher that z-index of [.googleAppsCont] */
+	z-index: 3;
+	/* This should be higher that z-index of [.googleAppsCont] */
 	transform-origin: top right;
 }
 
 /* Individual Icon Item Styling */
 .icon-item {
-	gap: 0.5rem; /* Item Gap*/
+	gap: 0.5rem;
+	/* Item Gap*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)